src/batMap.ml

```diff
@@ -53,6 +53,9 @@ module Concrete = struct
 
   let empty = Empty
 
+  let is_empty m =
+    m == Empty
```

Comment:
This should be just `= Empty`. In general we don't want to use physical equality `==` unless when absolutely necessary, because it has a tricky semantics that is less portable and needs more careful code reading.

Reply:
ok